modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -117,7 +118,7 @@ def to_faraday_upload(file_io, filename)
     end
 
     def process_response(response)
-      if response.success?
+      if response.success? || response.body.match?(NON_FAILING_ERROR_REGEX)
```

Comment:
Were you able to reproduce this situation? Maybe by doing to `PUT`s back to back? 

Probably a good candidate for a test either way

Reply:
Would definitely want to see a spec to exercise this code.

Reply:
Added a spec